samcli/commands/validate/validate.py

```diff
@@ -15,13 +17,25 @@
 from samcli.lib.telemetry.metric import track_command
 from samcli.cli.cli_config_file import configuration_option, TomlProvider
 from samcli.lib.utils.version_checker import check_newer_version
+from samcli.commands._utils.click_mutex import ClickMutex
 
+LOGGER = logging.getLogger("cfnlint")
+LOGGER.propagate = False
```

Comment:
Please use general way of instantiating the logger, one can be found here: https://github.com/aws/aws-sam-cli/blob/develop/samcli/lib/build/app_builder.py#L66

And why are you setting `propagate` to `False`?

Reply:
So the issue here is that cfnlint has its own loggers as well - if we use the general way, the logger wont catch the cfnlint logs. SAM cli also sets its root logger in main.py with basicConfig set to INFO level. This causes logs from cfnlint to be duplicated using that base logger. Setting propagate to false will prevent the duplicate logs. Changing the level in main.py would also prevent that and generally loggers are defaulted to WARNING, but im not sure how that would affect other logging behaviour in sam cli.